SteamKit2/SteamKit2/Steam/Handlers/SteamUser/SteamUser.cs

```diff
@@ -111,6 +111,12 @@ public sealed class LogOnDetails
             /// </summary>
             /// <value>The machine name.</value>
             public string? MachineName { get; set; } = $"{Environment.MachineName} (SteamKit2)";
+            /// <summary>
+            /// Gets or sets the chat mode.
+            /// Setting this field to 2 enables support for new Steam group chat.
```

Comment:
This sounds more like an enum than a uint.

Reply:
You can remove this string now that it's actually an enum.